Python/Tests/DjangoUITests/DjangoProjectUITests.cs

```diff
@@ -186,7 +191,7 @@ public class DjangoProjectUITests {
             Assert.IsNotNull(project.ProjectItems.Item("NewPage.html"));
         }
 
-        public void StartNewAppDuplicateName(VisualStudioApp app) {
+        public void StartNewAppDuplicateName(VisualStudioApp app, DjangoInterpreterSetter interpreterSetter) {
```

Comment:
I don't see where this parameter is used.

Reply:
It's instantiated via reflection by HostedPythonToolsTestRunner:
https://github.com/Microsoft/PTVS/blob/master/Common/Tests/Utilities.UI/HostedPythonToolsTestRunner.cs#L120